### PR TITLE
Week 2 Assignments

### PR DESCRIPTION
**Pull request by anugrahpaul2000**
Assignment 1 & 2 Finished

### CODE REVIEW

File: Week-2/Assignment-1/index.html [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+                >
+                  Read Article
+                </a>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="footer">
```
Comment (subhadip-pesto) [inline Week-2/Assignment-1/index.html]: don't put this form inside footer, and use footer tag

File: Week-2/Assignment-1/index.html [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+              <label class="fontMedium floatingLabel" for="confirmPassword"
+                >Confirm Password</label
+              >
+            </div>
+
+            <div class="floatInputGroup">
+              <select
+                id="country"
+                class="fontMedium floatingSelect"
+                onclick="this.setAttribute('value', this.value);"
```
Comment (subhadip-pesto) [inline Week-2/Assignment-1/index.html]: why you have added this onClick event?

File: Week-2/Assignment-1/style.css [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+}
+.floatingLabel a {
+  text-decoration: underline;
+  margin: 0px;
+  padding: 0px;
+}
+.floatingInlineInputGroup {
+  margin: 0px 5px;
+  display: flex;
+  justify-content: center;
```
Comment (subhadip-pesto) [inline Week-2/Assignment-1/style.css]: Floating Labels Should be left aligned, Keeping labels in the center is not a good UX.

File: Week-2/Assignment-2/style.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+}
+.tranformBottomLeft{
+  transform-origin: bottom left;
+}
+.tranformBottomRight{
+  transform-origin: bottom right;
+}
+
+#main:hover #topLeft {
+  rotate: 90deg;
```
Comment (subhadip-pesto) [inline Week-2/Assignment-2/style.css:65]: put this rotate, in transformTopLeft, and for the rest do the same.

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

File: Week-2/Assignment-1/index.html (hunk 1 above)
Comment (anugrahpaul2000) [reply Week-2/Assignment-1/index.html]: Done Sir

File: Week-2/Assignment-1/index.html (hunk 2 above)
Comment (anugrahpaul2000) [reply Week-2/Assignment-1/index.html]: Was using on click event to modify Select tag & make it look beautiful but removed that feature and forgot to remove the event.

File: Week-2/Assignment-1/style.css (hunk 3 above)
Comment (anugrahpaul2000) [reply Week-2/Assignment-1/style.css]: Ok sir. but this a sub div within a div. Align Left and center isn't actually doing anything.

File: Week-2/Assignment-2/style.css (hunk 4 above)
Comment (anugrahpaul2000) [reply Week-2/Assignment-2/style.css:65]: Tried Sir but animation breaks.

**Comment by anugrahpaul2000** [comment]
@subhadip-pesto Sir Please Review Once again.

**Comment by anugrahpaul2000** [comment]
@subhadip-pesto Updated as per 1-on-1 session Discussion

**Review by subhadip-pesto — APPROVED** [review]

**Pull request merged**